project-base/storefront/components/Pages/Registration/registrationFormMeta.tsx

```diff
@@ -90,19 +90,19 @@ type RegistrationFormMetaType = {
         successAndLogged: string;
     };
     fields: {
-        [key in keyof Omit<RegistrationFormType, 'passwordFirst' | 'passwordSecond'>]: {
+        [key in keyof Omit<RegistrationFormType, 'password' | 'passwordConfirm'>]: {
             name: key;
             label: string | JSX.Element;
             errorMessage: string | undefined;
         };
     } & {
-        passwordFirst: {
-            name: 'passwordFirst';
+        password: {
+            name: 'password';
             label: string;
             errorMessage: string | undefined;
         };
-        passwordSecond: {
-            name: 'passwordSecond';
+        passwordConfirm: {
+            name: 'passwordConfirm';
             label: string;
             errorMessage: string | undefined;
         };
```

Comment:
I would just remove these extra definitions
```ts
type RegistrationFormMetaType = {
    formName: string;
    messages: {
        error: string;
        successAndLogged: string;
    };
    fields: {
        [key in keyof RegistrationFormType]: {
            name: key;
            label: string | JSX.Element;
            errorMessage: string | undefined;
        };
    };
};
```

and then go to PasswordInputControlled.tsx:13 and change it to `label: ReactNode;`

the `TextInput` component which is used internally in `PasswordInputControlled` already accepts label as a ReactNode, so this is the only thing that has to be changed.